pages/_document.tsx

```diff
@@ -216,18 +218,14 @@ class MyDocument extends Document {
           <script async type="text/javascript" src="/static/scripts/form-polyfills.js"></script>
           {GoogleTagScript}
         </CustomHead>
+        <noscript>
+          <style type="text/css">{noJsScss}</style>
+          <JSDisabled
+            formProps={this.props.__NEXT_DATA__.props.pageProps.formRecord}
```

Comment:
It looks like the `JSDisabled` component only uses a logo out of the formRecord object. What if we rename `formProps` into `brandLogo` and only pass the relevant information?